apps/crunchy-postgres/workload/test.yml

```diff
@@ -123,7 +135,7 @@
             replace: "{{ lookup('env','PARALLEL_TRANSACTION') }}"           
         
         - name: Create Crunchy-postgres Loadgen Job
-          shell: kubectl apply -f {{ crunchy_loadgen }} -n {{ namespace }} 
+          shell: kubectl create -f {{ crunchy_loadgen }} -n {{ namespace }} 
```

Comment:
will it make any change ?

Reply:
The {{ crunchy_loadgen }} job here uses the generateName API which for which `apply` is not supported! (https://github.com/openebs/litmus/blob/master/apps/crunchy-postgres/workload/crunchy_postgres_loadgen.yml#L5)